src/main/java/seedu/address/model/policy/Policy.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.policy;
+
+import java.util.Objects;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Represents a person's policy in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Policy {
+
+    private final PolicyNumber policyNumber;
+    private final PolicyDate policyIssueDate;
+    private final PolicyDate policyExpiryDate;
```

Comment:
Should PolicyDate be changed to just using LocalDate type instead?

Reply:
Using PolicyDate allows us to control the behaviour of the object, and ensures it is immutable